packages/common/package.json

```diff
@@ -41,14 +41,18 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cumulus/aws-client": "1.19.0",
```

Comment:
Do we need this? I have a feeling this leads to a circular dependency, but can't remember. At any rate, I don't see this used in common

Reply:
Isn't it used in ... https://github.com/nasa/cumulus/blob/master/packages/common/aws.js

*oh*. It's deprecated in favor of it.     Missed that in grepping.   😅

Reply:
It *did* manage to build inexplicably though.

Reply:
00a44f857